OpenRA.Mods.RA/Widgets/Logic/ObserverStatsLogic.cs

```diff
@@ -304,6 +307,13 @@ string AverageEarnedPerMinute(double earned)
 			return "$" + (world.WorldTick == 0 ? 0 : earned / (world.WorldTick / 1500.0)).ToString("F2");
 		}
 
+		string KillDeathRatio(int killed, int dead)
+		{
+			var kdr = (float)killed;
+			if (dead > 0) kdr /= (float)dead;
```

Comment:
@Phrodoh I actually had it different before but then I noticed this style further down in the same file (currently mobile so hard to look up).

@chrisforbes thanks, will probably change it that way then.